### PR TITLE
show a banner when using Run From Package

### PR DESCRIPTION
**Pull request by ankitkumarr**

### CODE REVIEW

File: Kudu.Core/Helpers/EnvironmentHelper.cs [hunk 1]
```diff
@@ -31,11 +32,17 @@ public static bool IsWindowsContainers()
             return isolation == "hyperv" || isolation == "process";
         }
 
+        public static bool IsRunFromPackage()
+        {
+            string runFromPackage = System.Environment.GetEnvironmentVariable("WEBSITE_RUN_FROM_PACKAGE");
+            return !string.IsNullOrEmpty(runFromPackage) && !StringUtils.IsFalseLike(runFromPackage);
```
Comment (suwatch) [inline Kudu.Core/Helpers/EnvironmentHelper.cs:38]: also OR with Uri.TryParse - it could be URI format as well.

Comment (ankitkumarr) [reply Kudu.Core/Helpers/EnvironmentHelper.cs:38]: Sounds good. Right now, it checks if `WEBSITE_RUN_FROM_PACKAGE` exists and is not set to `0` or `false`, then we assume it's set to a correct value.
But I can add an explicit URL check as you suggest.

**Comment by ankitkumarr** [comment]
I will update the PR with screenshots of how this looks in kudu shortly.

**Comment by ankitkumarr** [comment]
I took the text from Portal. This text is also what Azure Portal shows when trying to modify files.
## Picture from main page

kudurfp2



## Picture from Console
Kudu-rfp1

**Comment by suwatch** [comment]
Looks great - could you squash it and we will rebase and merge?

**Pull request merged**